lib/router/router.js

```diff
@@ -109,6 +126,17 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && (this.options.noop || routeInfo.urlInfo.query.noop)) {
```

Comment:
If no steps are provided, the only processing would be to apply the "format" step. If your concern is that you don't want the processed image to be saved to cache, I would agree it should not be saving the processed image if you specify `useOriginal`.

So we want an option that says "if enabled, and no image steps are provided, respond back with original". I think we could achieve that by adding `useOriginal` to `options`, and making the changes to support that option. That way the format can still be processed, but it'll always serve the original if no image operations are specified. That seems reasonable.